documentation/src/main/asciidoc/public/getting-started/orm/index.adoc

```diff
@@ -80,7 +80,7 @@ which allows indexing and searching these documents.
 Below are the most common setups and matching dependencies for a quick start;
 read xref:{reference-documentation-location}#architecture[Architecture] for more information.
 
-Hibernate ORM + Lucene::
+Hibernate ORM + xref:{reference-documentation-location}#backend-lucene[Lucene]::
```

Comment:
maybe let's do the same for the standalone mapper as well ? ( ` Standalone POJO Mapper + Lucene:: `)

Reply:
Right, will do